openedx_events/content_authoring/signals.py

```diff
@@ -0,0 +1,23 @@
+"""
+Signal definitions for events within the architecture subdomain ``content_authoring``.
+
+All signals defined in this module must follow the name and versioning
+conventions specified in OEP-41.
+
+They also must comply with the payload definition specified in
+docs/decisions/0003-events-payload.rst
+"""
+
+from openedx_events.content_authoring.data import CourseCatalogData
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# .. event_type: org.openedx.content_authoring.course.catalog_info.changed.v1
+# .. event_name: COURSE_CATALOG_INFO_CHANGED
+# .. event_description: Fired when a course changes in Studio in a way that is relevant for Catalog consumers.
```

Comment:
1. [nit]: Questioning capitalization of "Catalog".
2. [questions] Should docs also refer to CMS and not Studio?

Reply:
I used 'Studio' since that is the name of the authoring service, while 'CMS' is the name of the full web service. It's a pretty delicate  distinction. https://github.com/openedx/edx-platform/blob/master/docs/decisions/0013-cms-vs-studio.rst

Reply:
marking as resolved since i think i answered the question